web/static/ivre/controllers.js

```diff
@@ -328,9 +328,11 @@ ivreWebUi
 	$scope.results = [];
 	$scope.display_mode = "host";
 	$scope.display_mode_args = [];
-	$scope.script_display_mode_needed_scripts_group = function(scripts) {
+	$scope.script_display_mode_needed_scripts_group = function(scripts, vulns) {
 	    if(scripts === undefined || scripts.length === 0)
 		return false;
+	    if(vulns !== undefined && vulns && !scripts.some(function (x) { return x.hasOwnProperty('vulns'); }))
```

Comment:
@p-l- Looking a second time on this one, `vulns === true` should suffice  and it would be shorter than `vulns !== undefined && vulns`. Which one do you prefer? I will redo the PR, sorry about that.

Reply:
Both are fine to me. Your code, your call! ;-)